torch/quantization/_quantize_script.py

```diff
@@ -79,13 +79,6 @@ def convert_script(model, inplace=False):
     if not inplace:
         model = model.copy()
     torch._C._jit_pass_insert_quant_dequant(model._c, 'forward', True)
-    if 'fbgemm' in torch.backends.quantized.supported_engines:
-        torch._C._jit_pass_insert_prepack_unpack(model._c)
```

Comment:
I think our fusion patterns expect `unpack` nodes to be present and with this change we are no longer inserting them. How does it work?

Reply:
oh, sorry, I'll keep prepack_unpack